src/blocks/Hero/Hero.js

```diff
@@ -90,14 +95,30 @@ function Hero(props) {
       <HeroMain>
         <HeroHeading>{heading}</HeroHeading>
 
-        {excerpt && <SanityHtml textBlocks={excerpt} />}
+        {excerpt && (
+          // Typography instead because SanityHTML does not work (?)
+          <Typography
```

Comment:
see comment about `HeroHeading`, would it be possible to create a look for `h3`, (h3 since variant determines the styling) to clean up the extra styling in the sx

Reply:
Semanticly, the excerpt should not be a heading.